src/main/resources/interface-specification.yml

```
@@ -6873,6 +6873,105 @@ paths:
               schema:
                 type: integer
               description: Time interval in milliseconds. Allowed requests will be constantly replenished during the interval. At the end of the interval the max allowed requests will be available
+  '/institutions/{institutionId}/products':
```

Just to avoid future endpoints collisions
```suggestion
  '/selfcare/institutions/products':
```